vrecord

```diff
@@ -1010,9 +1015,15 @@ else
 fi
 
 if [[ "${runtype}" = "passthrough" ]] ; then
-    "${FFPLAY_DECKLINK}" -v info -nostats "${GRAB_DECKLINK[@]}" 2> /tmp/vrecord_input.log \
+    if [[ "${media_player_choice}" = "ffplay" ]] ; then
```

Comment:
I suggest using `if [mpv] xxx ; else yyy ; fi` rather than an if/elif with no else. In this case if the variable somehow does not equal ffplay or mpv then it will only exit. If using if/else then you could also skip `media_player_choice="ffplay"` as that would be covered by the `else`.